library/unwind/src/wasm.rs

```diff
@@ -59,7 +59,7 @@ pub unsafe fn _Unwind_RaiseException(exception: *mut _Unwind_Exception) -> _Unwi
             wasm_throw(0, exception.cast())
         } else {
             let _ = exception;
-            core::arch::wasm32::unreachable()
+            core::arch::wasm::unreachable()
```

Comment:
[core::arch::wasm requires the simd_wasm64 feature](https://doc.rust-lang.org/nightly/core/arch/wasm/index.html), but AFAIK it isn't set in this crate.

https://github.com/rust-lang/rust/blob/963844b787f0f120547a01807dd5e67f2986c9a3/library/unwind/src/lib.rs#L1-L13

Reply:
Hm, I will close this PR in favor of #122797 then. Though IMO it's weird that `unreachable` on WASM64 requires `simd_wasm64` feature.

Reply:
Is that a mistake in `core::arch` or does that intrinsic really require SIMD, but only on wasm64 and not wasm32?

Reply:
@RalfJung I assume `simd_wasm64` really just means "special architecture intrinsics and features for wasm64" because... people just see `core::arch` and their brain shuts off kinda. I mean, all the architectural extension features were named `stdsimd` once.